hack/release.toml

```diff
@@ -14,6 +14,32 @@ preface = """\
 """
 
 [notes]
+    [notes.api-access-from-kubernetes]
+        title = "Talos API access from Kubernetes"
+        description = """\
+Talos now supports access to its API from within Kubernetes. It can be configured in the machine config as below:
+```yaml
+machine:
+  features:
+    kubernetesTalosAPIAccess:
+      enabled: true
+      allowedRoles:
+        - os:reader
+      allowedKubernetesNamespaces:
+        - kube-system
+```
+
+This feature introduces a new custom resource definition, `serviceaccounts.talos.dev`,
+which can be created by the user with the Talos roles they require.
```

Comment:
I think this needs to be clarified. We cant users to create resources of this type, not CRD itself.

Reply:
Ah yes, let me try to rephrase this.

Reply:
Corrected it and hopefully made it more clear.